Don’t run unit test in docker container in CI

geigerzaehler opened the pull request:
- [ ] Verify that code coverage works
- [ ] Update repository status checks to include `test-unit` and remove `test`

sleidig reviewed (commented):
Nice! :heart_eyes:    This looks so much simpler.

We still need to also change the `on-master-push.yml` workflow also.

And somehow an orphaned "test (linux/amd64, ubuntu-latest)Expected — Waiting for status to be reported" is still hanging around?

geigerzaehler commented:
> We still need to also change the `on-master-push.yml` workflow also.

Ah, yes, of course. Fixed it

> And somehow an orphaned "test (linux/amd64, ubuntu-latest)Expected — Waiting for status to be reported" is still hanging around?

Yeah, you need to update it again to "pull-request-update-or-push-tag / test-unit (pull_request)" because I changed the name to "test-unit" and it’s not a matrix build anymore
